libs/langchain/langchain/chat_models/databricks.py

```diff
@@ -0,0 +1,47 @@
+import logging
+from typing import List, Optional
+
+from langchain_core.pydantic_v1 import BaseModel, Extra
+
+from langchain.chat_models.mlflow import ChatMlflow
+
+logger = logging.getLogger(__name__)
+
+
+# Ignoring type because below is valid pydantic code
+# Unexpected keyword argument "extra" for "__init_subclass__" of "object"  [call-arg]
+class ChatParams(BaseModel, extra=Extra.allow):  # type: ignore[call-arg]
+    """Parameters for the `MLflow` LLM."""
+
+    temperature: float = 0.0
+    n: int = 1
+    stop: Optional[List[str]] = None
+    max_tokens: Optional[int] = None
+
+
+class ChatDatabricks(ChatMlflow):
+    """`Databricks` chat models API.
+
+    To use, you should have the ``mlflow`` python package installed.
+    For more information, see https://mlflow.org/docs/latest/llms/deployments/databricks.html.
+
+    Example:
+        .. code-block:: python
+
+            from langchain.chat_models import ChatDatabricks
+
+            chat = ChatDatabricks(
+                target_uri="databricks",
+                endpoint="chat",
+                params={"temperature": 0.1},
+            )
+    """
+
```

Comment:
Is it possible to have a `target_uri` that is not "databricks" for ChatDatabricks? If not, can we just set `target_uri` as a constant so that customers don't need to pass it every time?

```suggestion
target_uri: Literal["databricks"] = "databricks"
```

Essentially, `ChatDatabricks(endpoint=xxx)` is just an alias to `ChatMlflow(target_uri="databricks", endpoint=xxx)`.

Reply:
It's possible. A URI containing a profile: `databricks://<profile>`.

Reply:
Yes, but let's still give it a default value so that most customers don't need to set it every time.

Reply:
Ah I see

Reply:
Overriding `target_uri` makes sense.

Reply:
updated

Reply:
Yeah, that's why we are adding it just to `ChatDatabricks`(and `DatabricksEmbeddings`), not `ChatMlflow`.